packages/react-components/src/components/chart/types.ts

```diff
@@ -109,6 +109,7 @@ export type ChartOptions = {
   theme?: string;
   id?: string;
   titleText?: string;
+  onChartOptionsChange?: (options: Omit<ChartOptions, 'queries'>) => void;
```

Comment:
hmmmm, this needs to change ??

Reply:
<img width="739" alt="Screenshot 2024-01-11 at 10 46 53 AM" src="https://github.com/awslabs/iot-app-kit/assets/139960352/758af5f9-4c7d-4241-b7c4-581405f0bd42">

As per the story, I added it, I think in the future we may need to change some other chart options using the same callback but ok, for now, it is unnecessary to use all chart options I will update it.